pkg/plugins/commands.go

```diff
@@ -11,14 +13,50 @@ import (
 
 func Commands(l logger.Interface, s schema.Stage, fs vfs.FS, console Console) error {
 	var errs error
+	var wg sync.WaitGroup
+	wg.Add(len(s.Commands))
```

Comment:
how this is not breaking? you allocate by len(s.Commands) here, but you don't always call wg.Done. only when it's in the background

Reply:
ah I see, the run-type is for all the commands, I thought was one-by-one